Handbook: Working groups (#32278)
fleetdm/fleet: handbook/company/product-groups.md

> The [Slack channel](https://fleetdm.slack.com/archives/C097P4TAPRR), [kanban board](https://github.com/orgs/fleetdm/projects/92), and [GitHub label](https://github.com/fleetdm/fleet/labels?q=%23g-website) for this product group is `#g-website`.

## Working groups

Like a product group at Fleet, a working group is an arrangement of people from different functions. What makes a working group unique is that it is tasked with achieving a high-impact business goal fast. A working group disbands when the goal is achieved so Fleet stays fast and avoids unnecessary process.

Working groups are for important work that needs to be done quickly, when asynchronous work would be too slow.

Working groups should have their own section here in the handbook that includes the name, goal, resources, and contributors. See the commented out section in the handbook for an example.

Items to cover in the section:
- Name of the working group
- Desired business outcomes (goals)
- Link to the working group Slack channel and GitHub project
- Who is involved. This should include who the DRI is.
- Timeline. When will the working group start? When do we think we'll be done by?

### UX group

> The goals of the UX group are to catch the Fleet product up with the [latest brand colors/styles](https://github.com/fleetdm/confidential/issues/11765) from the website (fleetdm.com), address [UX debt (spiffier)](https://github.com/fleetdm/confidential/issues/11868), and ship the Fleet product's first [2 graphs](https://github.com/fleetdm/confidential/issues/11864).

Reviewer: ```suggestion
The goal of the UX group is to catch the Fleet product up with the [latest brand colors/styles](https://github.com/fleetdm/confidential/issues/11765) from the website (fleetdm.com), address [UX debt (spiffier)](https://github.com/fleetdm/confidential/issues/11868), and ship the Fleet product's first [2 graphs](https://github.com/fleetdm/confidential/issues/11864).
```

This group will kick off on 2025-08-27 and plans to wrap up on 2025-09-29.

| Responsibility                    | Human(s)                  |
|:----------------------------------|:--------------------------|
| Product Designer                  | [Mike Thomas](https://www.linkedin.com/in/mike-thomas-52277938) _([@mike-j-thomas](https://github.com/mike-j-thomas))_

> | Product Manager (DRI)             | [Noah Talerman](https://www.linkedin.com/in/markolisica/) _([@noahtalerman](https://github.com/marko-lisica))_

Author: ```suggestion
| Product Manager (DRI)             | [Noah Talerman](https://www.linkedin.com/in/noah-talerman/) _([@noahtalerman](https://github.com/noahtalerman))_
```

> | Product Manager (DRI)             | [Noah Talerman](https://www.linkedin.com/in/markolisica/) _([@noahtalerman](https://github.com/marko-lisica))_

Reviewer: This appears to link to Marko.

> [Janis Watts](https://www.linkedin.com/in/janis-watts-b080ab94/) _([@jmwatts](https://github.com/jmwatts))_

Reviewer: Should be a separate responsibility called "Quality Assurance" to match the format on the product groups' tables.

> | Developer                         | [Rachel Perkins](https://www.linkedin.com/in/rachelelysia/) _([@rachelelysia](https://github.com/rachelelysia))_

Reviewer: ```suggestion
| Developer                         | [Rachel Perkins](https://www.linkedin.com/in/rachelelysia/) _([@rachelelysia](https://github.com/rachelelysia))_, Luke Heath](https://www.linkedin.com/in/lukeheath/) _([@lukeheath](https://github.com/lukeheath))_
```

Reviewer: Volunteering to provide additional development support. ✨
